app/controllers/api/post_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::PostsController < ApplicationController
+  def index
+    user = User.find(params[:user_id])
+    posts = user.posts.order(Arel.sql('created_at DESC'))
+    render json: { posts: }, status: :ok
```

Comment:
- [ ] Request for all posts in a JSON format using RESTful API are not working. That goes for comments. Please kindly fix these issues. I didn't get any response to this GET request: http://localhost:3000/api/users/:user_id/posts or http://localhost:3000/api/users/:user_id/posts/:id/comments